_posts/2020-03-24-dask-developer-workshop.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title:  "Karlsruhe to D.C. ― a Dask story"
+date:   2020-03-24 20:00:00 +0100
+tags: technology python data-engineering Dask distributed
+header:
+  overlay_image: assets/images/dask_icon.svg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Lucas Rademaker & Nefta Kanilmaz
+author_profile: true
+---
+# Karlsruhe to D.C. ― a Dask story
+
+Back in February 2020, we (Florian Jetter, Nefta Kanilmaz and Lucas Rademaker) travelled to the Washington D.C. metropolitan area to attend the first Dask developer conference. Our primary goal was to discuss Blue Yonder’s issues related to Dask with the attending developers and users. When we returned back to Germany, we had not only connected with many of the core developers in the Dask community, but also had an (almost finished) implementation of a distributed semaphore in our bags. 
+
+There is [a blog post](https://blog.dask.org/2020/04/28/dask-summit) giving an overview on and summary of the workshop talks. We want to share our experience around this workshop. 
```

Comment:
"... a general overview ~~on~~ and summary of.... This blog post is a summary of _our_ experience of the workshop."

```suggestion
There is [a blog post](https://blog.dask.org/2020/04/28/dask-summit) giving a general overview and summary of the workshop talks. This blog post is a summary of _our_ experience of the workshop.. 
```